lib/genome/importers/tempus/tempus_tsv_importer.rb

```diff
@@ -0,0 +1,27 @@
+module Genome
+  module Importers
+    module Tempus
+      def self.source_info
+        {
+          base_url:           'https://www.tempus.com/',
+          site_url:           'https://www.tempus.com/',
+          citation:           'https://www.tempus.com/',
```

Comment:
It would be nice (if available) to link to a publication of theirs. Are there any from https://www.tempus.com/publications/ that describe the gene list? If not, perhaps we can cite the publications page here instead of the homepage.

Reply:
There is a publication about validation of this panel: https://www.tempus.com/clinical-validation-of-the-tempus-xt-next-generation-targeted-oncology-sequencing-assay/

This would be the link to use for the citation link, correct? Should anything be changed in the base or site urls (aside from the codeblocks mentioned below)?